purchase_order_report_selection/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Choice of columns for printing",
+    "summary": "Purchase order report selection, choice of columns for printing",
+    "author": "Cetmix,Odoo Community Association (OCA)",
```

Comment:
"author": "Ooops404, Cetmix, Odoo Community Association (OCA)",

Reply:
ok, I'll fix it